demos/gaze_estimation_demo/src/ie_wrapper.cpp

```diff
@@ -84,8 +84,8 @@ void IEWrapper::setInputBlob(const std::string& blobName,
     if (dimsProduct != data.size()) {
         throw std::runtime_error("Input data does not match size of the blob");
     }
-    auto inputBlob = request.GetBlob(blobName);
-    auto buffer = inputBlob->buffer().as<InferenceEngine::PrecisionTrait<InferenceEngine::Precision::FP32>::value_type *>();
+    LockedMemory<void> blobMapped = as<MemoryBlob>(request.GetBlob(blobName))->rwmap();
+    auto buffer = blobMapped.as<PrecisionTrait<Precision::FP32>::value_type *>();
```

Comment:
Might as well change the type to `float`...

Reply:
@IRDonch 

I agree that this change will make the code less complicated, but shouldn't we use PrecisionTrait<Precision::FP32>::value_type everywhere instead of direct type names?

What if we change the type behind "FP32" sometime in the future? This is unlikely, but still possible.

Reply:
It's unlikely FP32 is going to change. IE just needs that type for functions like `InputInfo::setPrecision()`.